libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -187,6 +235,12 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
             throw new Error('[columns] accepts numbers between 1 - 12');
         }
 
+        if (this._isColumnLayoutEnabled) {
+            this._breakpointObserver
+                .observe(this._responsiveBreakpointsService.getBreakpoints(this._responsiveBreakPointConfig))
+                .subscribe((result) => this._breakPointMeet(result));
+        }
```

Comment:
Let's hide `BreakpointObserver` into `ResponsiveBreakPointConfig` so others don't now about it
In this case it might look this:
```
this._responsiveBreakpointsService.observeBreakpointByConfig(this._responsiveBreakPointConfig)
    .subscribe(breakPointName => this._updateLayout(breakPointName))
```

Reply:
moved BreakpointObserver inside ResponsiveBreakPointService.